spec/features/workflow/drafting_requirements_spec.rb

```diff
@@ -1,56 +1,30 @@
 # frozen_string_literal: true
 
 RSpec.feature "Drafting requirements" do
-  scenario do
-    given_there_is_a_document_with_no_content
-    when_i_visit_the_edit_document_page
-    and_save_without_inputting_anything
-    then_i_should_see_an_error_message_regarding_a_missing_title
-
-    when_i_visit_the_edit_document_page
-    and_save_a_title_which_exceeds_the_max_length
-    then_i_should_see_error_messages_regarding_max_length
-
-    when_i_visit_the_edit_document_page
-    and_save_a_title_which_has_multiple_lines
-    then_i_should_see_error_messages_regarding_multiple_lines
-  end
-
-  def given_there_is_a_document_with_no_content
-    body_field_schema = build(:field_schema, id: "body", type: "govspeak")
-    document_type_schema = build(:document_type_schema, contents: [body_field_schema])
-    create(:document, document_type: document_type_schema.id, title: "")
-  end
+  include TopicsHelper
 
-  def when_i_visit_the_edit_document_page
-    visit edit_document_path(Document.last)
-  end
-
-  def and_save_without_inputting_anything
-    fill_in "document[title]", with: " "
-    click_on "Save"
-  end
-
-  def then_i_should_see_an_error_message_regarding_a_missing_title
-    expect(page).to have_content(I18n.t!("documents.edit.flashes.drafting_requirements.title_missing"))
-  end
+  scenario do
+    given_there_is_a_document
+    when_the_document_has_issues_to_fix
+    then_i_see_a_warning_to_fix_the_issues
 
-  def and_save_a_title_which_has_multiple_lines
-    fill_in "document[title]", with: "MULTIPLE LINES ARGHHH \r\n 2nd line"
-    click_on "Save"
+    #when_i_try_to_generate_a_preview
+    #then_i_see_an_error_to_fix_the_issues
```

Comment:
Are these meant to be commented out?

Reply:
I thought I'd write them here, as otherwise it didn't feel like a 'drafting requirements' spec. Would you prefer them left out?

Reply:
Yeah it looks a bit like a WIP to me with these commented out. I think the commit message would be the appropriate place to explain further info.

The presence of them confuses me so I'd suggest removing.

Reply:
OK. Done.